.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "flatpak-module-qpdf-pikepdf"]
+	path = flatpak-module-qpdf-pikepdf
+	url = ../../dreua/flatpak-module-qpdf-pikepdf.git
```

Comment:
this will not work.

Reply:
the submodule should be an https URL

Reply:
but then this shouldn't be used. Flathub doesn't control this which mean that if it disappear or is changed there is nothing we can do to even fix.